src/Components/Artwork/Details.tsx

```diff
@@ -141,8 +142,15 @@ export class Details extends React.Component<Props, null> {
     const inRunningAuction = sale && sale.is_auction && !sale.is_closed
 
     if (inRunningAuction) {
-      const sa = artwork.sale_artwork
-      return sa.highest_bid.display || sa.opening_bid.display
+      const highestBidDisplay = get(
+        artwork,
+        p => p.sale_artwork.highest_bid.display
+      )
+      const openingBidDisplay = get(
+        artwork,
+        p => p.sale_artwork.opening_bid.display
+      )
+      return highestBidDisplay || openingBidDisplay || ""
```

Comment:
Fixes https://sentry.io/artsynet/force-production/issues/731894886/

Reply:
since it's a regression it could use a test.